plugins/modules/ios_vlans.py

```diff
@@ -394,10 +430,249 @@
 # VLAN Type  SAID       MTU   Parent RingNo BridgeNo Stp  BrdgMode Trans1 Trans2
 # ---- ----- ---------- ----- ------ ------ -------- ---- -------- ------ ------
 # 1    enet  100001     1500  -      -      -        -    -        0      0
+# 10   enet  100010     1500  -      -      -        -    -        0      0
+# 20   enet  100020     610   -      -      -        -    -        0      0
+# 30   enet  100030     1500  -      -      -        -    -        0      0
 # 1002 fddi  101002     1500  -      -      -        -    -        0      0
 # 1003 tr    101003     1500  -      -      -        -    -        0      0
 # 1004 fdnet 101004     1500  -      -      -        ieee -        0      0
 # 1005 trnet 101005     1500  -      -      -        ibm  -        0      0
+#
+# Remote SPAN VLANs
+# ------------------------------------------------------------------------------
+# 10
+
+- name: Gather listed vlans with provided configurations
+  cisco.ios.ios_vlans:
+    config:
+    state: gathered
+
+# Module Execution Result:
+# ------------------------
+#
+# "gathered": [
+#         {
+#             "mtu": 1500,
+#             "name": "default",
+#             "shutdown": "disabled",
+#             "state": "active",
+#             "vlan_id": 1
+#         },
+#         {
+#             "mtu": 1500,
+#             "name": "VLAN0010",
+#             "shutdown": "disabled",
+#             "state": "active",
+#             "vlan_id": 10
+#         },
+#         {
+#             "mtu": 1500,
+#             "name": "VLAN0020",
+#             "shutdown": "disabled",
+#             "state": "active",
+#             "vlan_id": 20
+#         },
+#         {
+#             "mtu": 1500,
+#             "name": "VLAN0030",
+#             "shutdown": "disabled",
+#             "state": "active",
+#             "vlan_id": 30
+#         },
+#         {
+#             "mtu": 1500,
+#             "name": "fddi-default",
+#             "shutdown": "enabled",
+#             "state": "active",
+#             "vlan_id": 1002
+#         },
+#         {
+#             "mtu": 1500,
+#             "name": "token-ring-default",
+#             "shutdown": "enabled",
+#             "state": "active",
+#             "vlan_id": 1003
+#         },
+#         {
+#             "mtu": 1500,
+#             "name": "fddinet-default",
+#             "shutdown": "enabled",
+#             "state": "active",
+#             "vlan_id": 1004
+#         },
+#         {
+#             "mtu": 1500,
+#             "name": "trnet-default",
+#             "shutdown": "enabled",
+#             "state": "active",
+#             "vlan_id": 1005
+#         }
+#     ]
+
+# After state:
```

Comment:
Do we have to mention after state for gathered operation? IMO , we donot have to. What do you think?